x-pack/plugin/ent-search/qa/rest/roles.yml

```diff
@@ -2,6 +2,7 @@ admin:
   cluster:
     - manage_search_application
     - manage_behavioral_analytics
+    - manage_connector
```

Comment:
Can you add a comment on why this and below permissions are needed? What does it practically mean for users, if it does anything?